public/migrate.html

```diff
@@ -0,0 +1,161 @@
+          <textarea style="width: 100%" rows=30 id="entryArray" name="entryArray">
+          </textarea>
+        </label>
+      </fieldset>
+      <!-- 'featured' is set through administration -->
+      <fieldset>
+        <input type="hidden" id="nonce" name="nonce" value="">
+        <input type="hidden" id="csrf" name="csrfmiddlewaretoken" value="">
+        <button id="submit" disabled="disabled">submit</button>
+        <!-- this value SHOULD be set by the API server instead, but we do it here for now -->
+        <input type="hidden" id="user" name="submitted_by" value="">
+      </fieldset>
+    </form>
+
+  <script>
+
+    var currentNonce,
+        currentCSRF,
+        currentUser;
+      
+    var entries;
+    
+    //Sample:
+        //[{
+        //     "title": "test full entry 1",
+        //     "description": "description full entry",
+        //     "tags": ["tag1", "tag2"],
+        //     "interest": "interest field",
+        //     "get_involved": "get involved text field",
+        //     "get_involved_url": "http://example.com/getinvolved",
+        //     "thumbnail_url": "http://example.com/",
+        //     "content_url": "http://example.com/",
+        //     "internal_notes": "Some internal notes",
+        //     "featured": true,
+        //     "issues": ["Decentralization"],
+        //     "creators": ["Pomax", "Alan"]
+        // },
+        // {
+        //     "title": "test full entry 2",
+        //     "description": "description full entry",
+        //     "tags": ["tag1", "tag2"],
+        //     "interest": "interest field",
+        //     "get_involved": "get involved text field",
+        //     "get_involved_url": "http://example.com/getinvolved",
+        //     "thumbnail_url": "http://example.com/",
+        //     "content_url": "http://example.com/",
+        //     "internal_notes": "Some internal notes",
+        //     "featured": true,
+        //     "issues": ["Decentralization"],
+        //     "creators": ["Pomax", "Alan"]
+        // }];
+
+    // don't let the browser submit this form. We'll handle it.
+    document.forms.entries.onsubmit = () => false;
+
+    // as the API server for a nonce with associated csrf and user info
+    var getNonce = (callback) => {
+      var getnonce = new XMLHttpRequest();
+      getnonce.open("GET", "http://test.example.com:8000/nonce", true);
+      getnonce.onload = callback
+      getnonce.withCredentials = true
+      getnonce.send(null);
+    }
+
+    // put all the API server values in the form and enable submission
+    var confirmAuthentication = (evt) => {
+      var data = evt.target.response;
+      try {
+        data = JSON.parse(data);
+        currentUser = data.user;
+        currentNonce = data.nonce;
+        currentCSRF = data.csrf_token;
+
+        // when we have a nonce, set up actual form handling
+        var btn = document.querySelector("#submit");
+        btn.onclick = submitForm
```

Comment:
tiny user improvement: when you click "submit", grey out the submit button again, so you can't restart the submission process?